config/initializers/rollbar.rb

```diff
@@ -80,16 +80,5 @@
   # setup for Heroku. See:
   # https://devcenter.heroku.com/articles/deploying-to-a-custom-rails-environment
   config.environment = ENV['ROLLBAR_ENV'] || Rails.env
-
-  config.js_enabled = true
-  config.js_options = {
-    accessToken: ENV.fetch('ROLLBAR_CLIENT_ACCESS_TOKEN'),
-    captureUncaught: true,
-    captureUnhandledRejections: true,
-    logLevel: 'error',
-    hostWhiteList: ['coinfi.com', 'herokuapp.com'],
-    payload: {
-      environment: "production"
-    }
-  }
+  config.js_enabled = false
```

Comment:
Ah interesting, so the Rollbar `js_enabled` functionality doesn't work out of the box and requires custom integration?

Reply:
No, it works, but the problem was it was enabled for everything and couldn't be selectively disabled.
So, it was being injected into AMP pages, which can't support external javascript.